### PR TITLE
Remove console

### LINKED ISSUE #9: Remove console.*
We should avoid using `console.*` and  use a proper logger library that gets activated by a `verbose` or `debug` flag.

### PR DESCRIPTION
**Pull request by cef62**
This PR closes #9 

I've added `debug`, `chalk` and `node-emoji` to the project. Also I've updated current `console.log` statement to use `debug` instead.

All debug statements are grouped under `cmc:*` namespace, execute the module using `DEBUG=cmc:*` to enable all of them.

@MicheleBertoli feel free to improve the output and the emojis used :),  [Here](http://www.emoji-cheat-sheet.com/) you can find a partial list of supported emojis 👊

### CODE REVIEW

**Comment by cef62** [comment]
@MicheleBertoli seems that logs inside the ast traverse loop greatly slow the process, can you confirm that happens also to you?

**Comment by cef62** [comment]
Nevermind, now the process is fast as usual. Probably I had some process stuck on the shell.

File: src/update-imports.js [hunk 1]
```diff
@@ -75,8 +80,10 @@ const updateCssImports = (file, contentsMap, verbose = false) => {
         plugins: ['jsx', 'flow'],
       })
 
+      debug(`Search for  css-modules imports in: ${chalk.red.bold(file)}`)
```
Comment (MicheleBertoli) [inline src/update-imports.js]: Double space between "for" and "css-modules".

**Comment by MicheleBertoli** [comment]
Awesome, @cef62!
I ❤️ emojis :)

Just one small comment and it's good to go.

**Pull request merged**